localisation/00_County_names.csv

```diff
@@ -169,6 +173,8 @@ c_banana_forest;Banana Forest;Banana Forest;Banana Forest;;Banana Forest;;;;;;;;
 c_banana_forest_adj;Banana Forestian;;Banana Forester;;;;;;;;;;;x
 c_barbthorn;Barbthorn;Barbthorn;Barbthorn;;Barbthorn;;;;;;;;;x
 c_barbthorn_adj;Barbthornian;;barbthorner;;;;;;;;;;;x
+c_barren_atol;Barren Atol;Barren Atol;Barren Atol;;Barren Atol;;;;;;;;;x
+c_barren_atol_adj;Barren Atolian;;Barren Atoler;;;;;;;;;;;x
```

Comment:
Fixed spelling. Atollian and Atoller aren't exactly great adjectives, but I can't think of better ones, so left as is.
```suggestion
c_barren_atoll_adj;Barren Atollian;;Barren Atoller;;;;;;;;;;;x
```